lib/features/modeling/ElementFactory.js

```diff
@@ -127,6 +127,10 @@ ElementFactory.prototype.createBpmnElement = function(elementType, attrs) {
     di.isMarkerVisible = true;
   }
 
+  if (attrs.triggeredByEvent) {
+    businessObject.triggeredByEvent = attrs.triggeredByEvent;
```

Comment:
We should also delete `attrs` in this case, cf. handling of other special attributes.

Reply:
Also, consider adding a test case for this.